README.md

```diff
@@ -302,24 +302,30 @@ To use a new virtual environment, there are two steps you need to complete. The
 
 If you want to learn more about Conda environments, have a look at the [documentation](https://conda.io/docs/user-guide/tasks/manage-environments.html), otherwise, let’s give this a try.
 
-You need to start by navigating into the root of this project folder, so you’re going to want to type `cd  dsc-1-01-05-setting-up-environment` in your terminal if you didn't already.
+You need to start by navigating into the root of this project folder, so you’re going to want to type `cd  dsc-data-science-env` in your terminal if you didn't already.
 
 Then to create the environment, on a mac, type `conda env create -f environment.yml`. On windows, type `conda env create -f windows.yml`. Depending on the speed of your computer and your internet connection it may take up to five minutes for this to complete. While it does you should see output similar to that displayed below start to appear in your terminal.
 
 <img src='http://curriculum-content.s3.amazonaws.com/data-science/screen-47.png' width="750">
 
 If you see a message that states “WARNING: A newer version of Conda exists”, run `conda update -n base conda` and then try again to create the environment using `conda env create -f environment.yml`.
 
-Next, try activating the environment. Whether you're on a Mac or using git bash on a windows machine, type `source activate learn-env` (if you have an issue with running git bash, the command to activate Conda within the Conda shell on windows is `activate learn-env`).
+Next, try activating the environment. Whether you're on a Mac or using git bash on a windows machine, type `conda activate learn-env` (if you have an issue with running git bash, the command to activate Conda within the Conda shell on windows is `activate learn-env`).
 
 To confirm that it worked, type `conda info --envs` and confirm that the output in the terminal ends with /learn-env - e.g. *  /Users/peterbell/anaconda3/envs/learn-env
 
+## Setting your Default Environment
+
+You have successfully created your virtual environment! To be sure that you are using the learn-env it's helpful to set it as your default environment so that you don't need to remember to manually switch to it every time you open terminal.
+
+On a Mac, run `echo "conda activate learn-env" >> ~/.bash_profile` to add the configuration to your bash profile and then run `source ~/.bash_profile` to activate the changes you just made. 
+
 
```

Comment:
@sumedh10 could you confirm this is the right solution for setting a default environement on a windows machine https://stackoverflow.com/a/53929174

Reply:
@alexgriff  I am unable to edit the `activate.bat` file